uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -85,7 +86,7 @@ define([
       vm.loading.calendar = true;
       vm._loadContacts()
         .then(function () {
-          vm._loadLeaveRequestsAndCalender()
+          vm._loadLeaveRequestsAndCalendar()
             .then(function () {
               vm.loading.calendar = false;
```

Comment:
you have this `vm.loading.calendar = false` on both children, so it should be moved to the parent